nixos/modules/system/boot/systemd.nix

```diff
@@ -750,6 +669,13 @@ in
         config.boot.extraSystemdUnitPaths != [ ]
       ) "${builtins.concatStringsSep ":" config.boot.extraSystemdUnitPaths}:";
     };
+    systemd.settings.Manager = {
+      ManagerEnvironment = lib.concatStringsSep " " (
+        lib.mapAttrsToList (n: v: "${n}=${lib.escapeShellArg v}") cfg.managerEnvironment
+      );
```

Comment:
(I’d still like to see this RFC 42‐ized, but I agree that it can wait for another PR.)